<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="InequalityTest-20160406173612" tests="1" time="0.001">
	<testcase classname="InequalityTest" name="test_IsClassify_1" time="0.001">
		<error message="'InequalityTest' object has no attribute 'Ontobi_API'" type="AttributeError">
<![CDATA[Traceback (most recent call last):
  File "C:/Users/CHRIS/PycharmProjects/Requests/Requests.py", line 24, in test_IsClassify_1
    Actuval=self.Ontobi_API(text)
AttributeError: 'InequalityTest' object has no attribute 'Ontobi_API'
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
